### PR TITLE
Fix the missing parameters in the "Test in Docker" section in README.md

### LINKED ISSUE #1243: Fix missing command parameters of "Test in Docker" section in README.md

### PR DESCRIPTION
**Pull request by brightcoder01**
Fix #1243

### CODE REVIEW

File: elasticdl/README.md [hunk 1]
```diff
@@ -105,8 +105,9 @@ docker run --rm -it -v $PWD:/work -w /work \
 
 In a terminal, start master to distribute mnist training tasks.
 
-```
-docker run --net=host --rm -it elasticdl:dev \
+```bash
+docker run --net=host --rm -it -v $EDL_REPO:/v -w /v \
```
Comment (skydoorkai) [inline elasticdl/README.md]: Use a more meaningful directory name than `/v`, such as `/edl_dir`

File: elasticdl/README.md [hunk 2]
```diff
@@ -123,12 +124,15 @@ docker run --net=host --rm -it elasticdl:dev \
 
 In another terminal, start a worker
 
-```
-docker run --net=host --rm -it elasticdl:dev \
+```bash
+docker run --net=host --rm -it -v $EDL_REPO:/v -w /v \
```
Comment (skydoorkai) [inline elasticdl/README.md]: Same here.

File: elasticdl/README.md (hunk 1 above)
Comment (brightcoder01) [reply elasticdl/README.md]: Good idea, updated.

File: elasticdl/README.md (hunk 2 above)
Comment (brightcoder01) [reply elasticdl/README.md]: Good idea, updated.

**Review by skydoorkai — APPROVED** [review]

**Pull request merged**